AutoRest/Generators/CSharp/Azure.CSharp/AzureCSharpCodeGenerator.cs

```diff
@@ -60,6 +60,7 @@ public override string ImplementationFileExtension
         /// <param name="serviceClient"></param>
         public override void NormalizeClientModel(ServiceClient serviceClient)
         {
+            Extensions.NormalizeClientModel(serviceClient, Settings);
```

Comment:
You shouldn't call Extensions.NormalizeClientModel here, since AzureExtensions.NormalizeAzureClientModel contains generic extensions as well.